src/main/java/org/radarcns/management/service/mapper/UserMapper.java

```diff
@@ -35,7 +36,9 @@
         if(roles == null) {
             return null;
         }
-        return roles.stream().map(role -> role.getAuthority().getName()).collect(Collectors.toSet());
+        return roles.stream()
+            .filter(Objects::nonNull)
```

Comment:
How can null end up in the roles? There probably must be a check somewhere else.

Reply:
Hmm, not sure, might have been related to the bug in UserService that caused the other non-null check in UserMapper to be necessary. Reverted this change.